pkg/notification/entities.go

```diff
@@ -49,3 +54,9 @@ type ProjectConfigUpdateNotification struct {
 	Type     Type
 	Revision string
 }
+
+type LogEventNotification struct {
+	Type         Type
+	LogEvent  	 *bytes.Buffer
```

Comment:
I think it may be better not to convert to bytes , and just pass LogEvent, then Marshal is not needed.

Reply:
cyclical import between event and notification packages.